crates/env/src/call/call_builder.rs

```diff
@@ -30,6 +30,7 @@ use core::marker::PhantomData;
 
 /// The final parameters to the cross-contract call.
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Not really happy with the solution of adding the `#[allow(dead_code)]` attributes, any other ideas?

Reply:
how about some proper `#[cfg(..)]`?

Reply:
I'd not like to accept an `#[allow(dead_code)]` hack-fix.

Reply:
> how about some proper `#[cfg(..)]`?

The `call_builder` is only used in examples and tests, but not in the crate itself ‒ that's why the fields are never read there. How would a `#[cfg(…)]` fix this? We run `clippy` on all targets.